cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperator.java

```diff
@@ -788,6 +941,7 @@ private KafkaRebalanceStatus buildRebalanceStatus(String sessionID, KafkaRebalan
                                                           boolean dryrun, RebalanceOptions.RebalanceOptionsBuilder rebalanceOptionsBuilder, String userTaskID) {
 
         log.info("{}: Requesting Cruise Control rebalance [dryrun={}]", reconciliation, dryrun);
+        rebalanceOptionsBuilder.withVerboseResponse();
```

Comment:
I remember that at the beginning we decided to make the verbosity optional using a `verbose` field in the `KafkaRebalance` resource. We removed that field for the first implementation. It should be the time to re-add the field to allow the user to decide instead of "hardcoding" that it's always verbose. Wdyt?

Reply:
We need the verbose option to obtain the load information. We could make the broker load optional in the rebalance, so if the user doesn't want it we don't request it.